docs/firstobject.md

```diff
@@ -109,41 +113,39 @@ As the name of this step suggests; you will be adding all your Linked Data eleme
 
 Again, enter all fields with a red `*` and when you are finished, click `Move on to next step`
 
-![Ingest Step 2 - Linked Data](images/firstobject/06_step-two_linked-data.jpg)
+![Ingest Step 3 - Linked Data](images/firstobject/07_step-three_linked-data.png){ loading=lazy }
 
-### Collections
+!!! tip
 
-Since this is our first digital object we do not yet have a Digital Object *Collection* for which `My First Digital Object` could be a member of. In other words, you can leave `Collection Membership` blank and click `Next: Upload Files`.
+    When entering a location, place or address you will need to click on the `Search OpenStreet Maps` button, which is what that big red arrow is pointing to in the screenshot below.
 
-![Ingest Step 3 - Collections](images/firstobject/07_step-three_collections.jpg)
+    ![Ingest Step 3 Tip - Linked Data](images/firstobject/07_linked-data_tip.png){ loading=lazy }
 
 ### Upload Files
 
 Now we will upload our `Photograph`. Click `Choose Files` to open your file selector window and choose which file you would like to ingest.
 
-![Ingest Step 4 - Attach Files pt 1](images/firstobject/08_step-four_upload-files_1.jpg)
+![Ingest Step 4 - Attach Files pt 1](images/firstobject/08_step-four_upload-files_1.png){ loading=lazy }
 
 Once you've uploaded your file, you will see all the Exif data extracted from the image, like so...
 
-![Ingest Step 4 - Attach Files pt 2](images/firstobject/09_step-four_upload-files_2.jpg)
+![Ingest Step 4 - Attach Files pt 2](images/firstobject/09_step-four_upload-files_2.png){ loading=lazy }
 
 Once you've mentally digested all of that data, let's go ahead and click `Save Metadata`.
 
-**Question:** "Wait! Don't I need to switch `Save as` from *Draft* to *Publish*? I'm ready for this Digital Object to be shared with the world!"
+??? question "Wait! Don't I need to switch `Status` from *Draft* to *Publish*? I'm ready for this Digital Object to be shared with the world!"
```

Comment:
@aksm but the screenshot says "Save as" ....

Reply:
@DiegoPino Meant to rephrase or reorder/restructure that one because "Save as" isn't available at that point so it might be confusing if following step-by-step. Will come back to this.